### DISCUSSION TITLE
Space ROS Technical Meeting Agenda 2024-01-11

### DISCUSSION THREAD

**Discussion by Bckempa** [space-ros/space-ros#124] [General]
Previous agenda: #122 

### Agenda
- [x] Introduce any new attendees.
- [x] Summarize any PRs merged since last meeting.
    - [ ] https://github.com/space-ros/docker/pull/97
    - [ ] https://github.com/space-ros/docker/pull/95
    - [ ] https://github.com/space-ros/docs/pull/23
    - [ ] https://github.com/space-ros/docs/pull/27
    - [ ] https://github.com/space-ros/docker/pull/84
- [ ] Review issues and PRs ([nominated for next release](https://github.com/orgs/space-ros/projects/3/views/6)).
- [ ] Open next meeting discussion

**Comment by ivanperez-keera** [comment]
I've edited the message above to include another PR merged.

Also closed:
- https://github.com/space-ros/space-ros/issues/57
- https://github.com/space-ros/space-ros/issues/41

I've assigned @EzraBrooks to four PRs, because they are very small (just minor changes to the READMEs):
- https://github.com/space-ros/docker/pull/118
- https://github.com/space-ros/docker/pull/117
- https://github.com/space-ros/docker/pull/116
- https://github.com/space-ros/docker/pull/115

Please, Ezra, assign someone else if you can't process those.

~~I've asked Brian to one PR. Same rationale: https://github.com/space-ros/docs/pull/27~~ (closed and moved up)

Please, Brian, assign someone else if you can't process those.

**Reply by EzraBrooks** [reply]
approved all of the above PRs, thanks for the work!

**Comment by Bckempa** [comment]
I'm looking to finish up some of the CI improvements:

- https://github.com/space-ros/space-ros/issues/80
- https://github.com/space-ros/space-ros/issues/90
- (maybe) https://github.com/space-ros/docker/issues/82

But I have a [question](https://github.com/space-ros/space-ros/discussions/114#discussioncomment-7968543) about doing these checks in GHAs vs in Earthly. TL;DR: I'd like to implement this in Earthly because I have some concerns about the community GHAs and it's easier to switch to them later if we dump Earthly.

**Comment by xfiderek** [comment]
this PR is open and ready for review for quite a while: https://github.com/space-ros/docker/pull/90

**Comment by ivanperez-keera** [comment]
I opened a discussion for the next agenda: #126 .
